addons/viewdistance/stringtable.xml

```diff
@@ -114,7 +114,7 @@
             <Portuguese>Distância de visão dinâmica dos objetos</Portuguese>
         </Key>
         <Key ID="STR_ACE_ViewDistance_object_Description">
-            <English>Sets the object view distance as a coefficient of the view distance.</English>
+            <English>Sets the object view distance as a coefficient of the view distance or based on field of view. FoV Based's lowest and highest value is the value which is set when enabling the option.</English>
```

Comment:
Not sure what the process is for changing strings, remove the rest or hunt for translators? :grimacing: